tasks/main.yml

```diff
@@ -8,10 +8,10 @@
 - block:
     - name: Login to oVirt
       ovirt_auth:
-        url: "{{ engine_url | default(omit) }}"
-        username: "{{ engine_user | default(omit) }}"
-        password: "{{ engine_password | default(omit) }}"
-        ca_file: "{{ engine_cafile | default(omit) }}"
+        url: "{{ engine_url | default(omit) | default(lookup('env','host')) }}"
```

Comment:
I think `default(omit)` should be last, did you test it?

Reply:
I got same results from it, but i'll look on it bit more.